appeals/web/src/server/lib/mappers/page-components/interested-party-comments/view-and-review/reject.mapper.js

```diff
@@ -0,0 +1,92 @@
+import { appealShortReference } from '#lib/appeals-formatter.js';
```

Comment:
Probably a merge issue but these mappers have moved back to the feature dir. Please make sure this PR won't reintroduce this file

Reply:
this file should have been removed, possibly bad rebase